sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -32,10 +33,13 @@
     private static final String INVALID_APPLICATION_ID_LENGTH = "'applicationId' length cannot be greater than "
         + MAX_APPLICATION_ID_LENGTH;
     private static final String INVALID_APPLICATION_ID_SPACE = "'applicationId' cannot contain spaces.";
-    private static final List<String> DEFAULT_HEADERS_WHITELIST = Arrays.asList(
+    private static final List<String> DEFAULT_HEADERS_ALLOWLIST = Arrays.asList(
+        "x-ms-request-id",
         "x-ms-client-request-id",
         "x-ms-return-client-request-id",
         "traceparent",
+        "MS-CV",
```

Comment:
what's `MS-CV` ?

Reply:
Not entirely certain but I saw .NET had it in their list and wanted to standardize with what they had.

@tg-msft, would you happen to know more about this header?

Reply:
https://stackoverflow.com/questions/57544515/what-is-ms-cv-in-response-header